### DISCUSSION TITLE
Show IMDb rating

### DISCUSSION THREAD

**Discussion by monoroc** [sct/overseerr#1893] [Feature Requests]
Is it possible to also show the IMDb rating on the movie detail page (next to the Rotten Tomatoes and TMDB score)?
As far as I know there's no official API for IMDB, but [OMDb API](https://www.omdbapi.com/) could provide you that data instead :)

**Comment by TheCatLady** [comment]
From #612:
> OMDB is limited to 1000 requests per day unless you become a patreon.

**Reply by monoroc** [reply]
Okay, so a hardcoded API key wouldn't work. But what if the user could enter his own OMDb API key in the settings and a new job would request missing ratings every 24h? Granted, the initial completion would probably take a couple days, but after that, the 1000 requests limit would be negligible.

**Reply by TheCatLady** [reply]
We do not save any media data in the DB; it is all fetched on demand. With the limit being so low, I imagine it'd be pretty easy to exhaust all 1000 allowed requests, especially if you have many users.

With things as they are, this isn't really feasible. We can always re-evaluate if there are changes to the API limit or other alternatives.

**Reply by monoroc** [reply]
Gotcha. I thought that metadata was stored locally. But yeah, if it's fetched dynamically, I don't see any other possibilities either. Thanks for clarifying!
